### PR TITLE
Packing nomenclature mods

### PR DESCRIPTION
**Pull request by WestLangley**
Packing nomenclature mods as proposed in #8602.

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl [hunk 1]
```diff
@@ -9,6 +9,6 @@ void main () {
 
 	#include <clipping_planes_fragment>
 
-	gl_FragColor = packLinearUnitToRGBA( length( vWorldPosition.xyz - lightPos.xyz ) / 1000.0 );
+	gl_FragColor = packDepthToRGBA( length( vWorldPosition.xyz - lightPos.xyz ) / 1000.0 );
```
Comment (bhouston) [inline src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl]: In this case the value being packed isn't a true depth, it is distance metric.

Comment (WestLangley) [inline src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl]: I saw that. I think the shader is misusing the method. The argument should be in [0, 1], no? That argument only works on scenes of certain size units.

Comment (bhouston) [inline src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl]: More details here: https://github.com/mrdoob/three.js/issues/8559

**Pull request merged**